### ISSUE TITLE
Add alt tag to spaghetti twirl image

### ISSUE THREAD

**Issue by OllieBullGB** [stevenaeola/progblack_2223#50]
There is currently no tag on the spaghetti twirl image, following WCAG guidelines images should have an alt attribute describing the image
